Makefile

```diff
@@ -56,31 +56,6 @@ test:
 	coverage3 report -m
 	[ -f "/usr/bin/coveralls" ] && [ -n "$(COVERALLS_REPO_TOKEN)" ] && coveralls || echo
 
-# need `losetup`, which needs Docker to be in privileged mode (--privileged)
-# but even so fails in Travis CI
-test_images:
-	sudo -E ./tests/test_cli.sh tests/cli/test_compose_ext4-filesystem.sh  \
-				    tests/cli/test_compose_partitioned-disk.sh \
-				    tests/cli/test_compose_tar.sh              \
-				    tests/cli/test_compose_tar_kickstart.sh    \
-				    tests/cli/test_compose_qcow2.sh            \
-				    tests/cli/test_compose_live-iso.sh
-
-test_aws:
-	sudo -E ./tests/test_cli.sh tests/cli/test_build_and_deploy_aws.sh
-
-test_azure:
-	sudo -E ./tests/test_cli.sh tests/cli/test_build_and_deploy_azure.sh
-
-test_openstack:
-	sudo -E ./tests/test_cli.sh tests/cli/test_build_and_deploy_openstack.sh
-
-test_vmware:
-	sudo -E ./tests/test_cli.sh tests/cli/test_build_and_deploy_vmware.sh
-
-test_cli:
```

Comment:
This test is actually useful when running in podman. I usually run `RUN_TESTS="ci test_cli" DOCKER=podman make test-in-docker` to test my changes locally.